Makefile

```diff
@@ -77,6 +77,15 @@ configgen:
 	echo "" >> conf/st2.conf.sample
 	. $(VIRTUALENV_DIR)/bin/activate; python ./tools/config_gen.py >> conf/st2.conf.sample;
 
+.PHONY: .generate-openapi-spec
```

Comment:
I am also fine if you decide to remove --generate from st2-validate-api-spec and create a new tool st2-generate-api-spec. Having something in tools/ sucks because they won't be shipped in packages. Sorry,  I know there are too many things going on but I expected us to argue about this :P